cumulusci/tasks/robotframework/robotframework.py

```diff
@@ -202,6 +204,26 @@ def _run_task(self):
             if prefix in source_paths:
                 self.options["suites"][i] = os.path.join(source_paths[prefix], path)
 
+        # this is necessary so that javascript-based keywords have access
+        # to at least some of the org info
+        cci_context = json.dumps(
+            {
+                "project_config": {
+                    "repo_name": self.project_config.repo_name,
+                    "repo_root": self.project_config.repo_root,
+                },
+                "org": {
+                    "name": self.org_config.name,
+                    "instance_url": self.org_config.instance_url,
+                    "org_id": self.org_config.org_id,
+                },
+            }
+        )
+        os.environ["CCI_CONTEXT"] = cci_context
```

Comment:
Remind me: does this persist beyond the Robot process invocation? We do not want to leak the access token in an env var.

Reply:
It should only live as long as the robot task is alive.